x-pack/docs/en/security/operator-privileges/operator-only-functionality.asciidoc

```diff
@@ -0,0 +1,29 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[operator-only-functionality]]
+=== Operator-only functionality
+Operator-only functionalities are APIs or dynamic cluster settings protected by operator privileges.
```

Comment:
I think "operator only functionality" is fine but functionalities (in plural) sounds a little weird to me. Maybe we could rephrase some of these sentences so that they are in the singular instead.

```suggestion
Operator privileges provides protection for APIs and dynamic cluster settings.
Any API or cluster setting that is protected by operator privileges is known as _operator-only functionality_
```